LeoProjectGaming/src/main/resources/application.properties

```diff
@@ -1 +0,0 @@
-
```

Comment:
Did you delete it or just left it empty?

Reply:
it should be deleted hmm